ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -182,6 +185,21 @@ router bgp {{ router_bgp.as }}
       route-target export {{ route_target }}
 {%             endfor %}
 {%         endif %}
+{%         if router_bgp.vlans[vlan].route_targets.import_evpn_domains is arista.avd.defined %}
+{%             for route_target in router_bgp.vlans[vlan].route_targets.import_evpn_domains %}
```

Comment:
Please remove above if condition and use natural_sort filter to make sure the consistency.
```for route_target in router_bgp.vlans[vlan].route_targets.import_evpn_domains | arista.avd.natural_sort ```

Reply:
Refactored to use natural_sort for all route-targets.